chapters/002-smol-internet/README.md

```diff
@@ -508,14 +423,9 @@ rtt min/avg/max/mdev = 0.080/0.172/0.264/0.092 ms
 
 > `boudi` running `tcpdump -ni eth1` <= `squasheeba` interface
 
-NOTHING shows up here! Why? When the packets reach `boudi`, `boudi` recognizes
-that they have reached their destination. There's no need to send the packet
-through the `squasheeba` interface just to stay on the same machine. The work is
-done!
+NOTHING shows up here! Why? When the packets reach `boudi`, `boudi` recognizes that they have reached their destination. There's no need to send the packet through the `squasheeba` interface just to stay on the same machine. The work is done!
```

Comment:
Oh sweet! We talked about this in Ch 4 as well. perhaps we ought to reference back to this lesson?